tw-tasks-jobs/src/main/java/com/transferwise/tasks/impl/jobs/JobsService.java

```diff
@@ -36,7 +36,7 @@ public class JobsService implements IJobsService, GracefulShutdownStrategy, Init
   @Autowired
   private ITasksService tasksService;
   @Autowired
-  private JobsProperties jobsProperties;
+  protected JobsProperties jobsProperties;
```

Comment:
I'd better move the `tw-tasks-jobs-test/src/main/java/com/transferwise/tasks/impl/jobs/test/TestJobsService.java` to `tw-tasks-jobs-test/src/main/java/com/transferwise/tasks/impl/jobs/TestJobsService.java` ?